docs/metrics/getting-started-histogram/README.md

```diff
@@ -22,7 +22,7 @@ Install the
 package:
 
 ```sh
-dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Exporter.Console --version 1.2.*.*
```

Comment:
The same would apply to other README.md files under the `/docs/metrics/` folder.

Reply:
In addition, shall we update the corresponding .csproj files to align with the version in the .md files?

https://github.com/open-telemetry/opentelemetry-dotnet/blob/64170d38650df55d3a05fae80b0674185c4125a8/docs/metrics/getting-started-histogram/getting-started-histogram.csproj#L4

Reply:
I think the `.csproj` files are fine since if you clone the repository, it will use your local files for this package